app/src/main/res/layout/locale_settings_item.xml

```diff
@@ -2,18 +2,18 @@
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content">
+    android:layout_height="48dp"
```

Comment:
@brampitoyo What is the UX preference here: all items have the same height or all have the same padding?

We currently use the same padding on all items so that the whitespace between each language is consistent even if the subtitle is visible or hidden.

Reply:
I've fixed it for "Follow device language" because without fixed heigh that bar will be 24dp and broke the design.
https://app.abstract.com/share/2e7f2625-24e4-41e9-bb9b-5e78c31dbbbe?mode=build&sha=62825ec262f4d071a8acae25bec9180f84bf4d3d

Reply:
![resim](https://user-images.githubusercontent.com/17825767/86032472-174fcb00-ba40-11ea-83b3-5950d7ff33a8.png)

Reply:
@hakkikaancaliskan @NotWoods Sorry I missed this particular problem.

It seems that all languages have subtitles, the only exception being “Follow device language”.

This means that the one outlier item on this page, “Follow device language”, should match the height of all the other languages.

(And sorry for not making the right margin 16dp – it should be. This is the same right margin that we use in other list items).